### PR TITLE
Test for user profile, image storage and restaurant occupancy

### PR DESCRIPTION
**Pull request by mbouilloud**
I wrote the missing tests for profile picture update, image storage DB and location listener for occupancy updates.

### CODE REVIEW

File: app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -3,7 +3,11 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.github.sdp.ratemyepfl">
 
-    <application tools:ignore="MissingApplicationIcon">
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```
Comment (jmettler27) [inline app/src/debug/AndroidManifest.xml:6]: DId you find a way to get rid of the check warning in this line?

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/utils/TestUtils.kt [hunk 2]
```diff
@@ -53,4 +67,46 @@ object TestUtils {
...
+        scenario.onActivity {
+            activity = it
+        }
+        return activity!!
+    }
+
+    /**
+     * Code to test gallery : https://proandroiddev.com/testing-camera-and-galley-intents-with-espresso-218eb9f59da9
+     */
+    fun savePickedImage(activity: Activity, resId: Int) {
```
Comment (jmettler27) [inline app/src/androidTest/java/com/github/sdp/ratemyepfl/utils/TestUtils.kt:85]: Very useful

**Review by jmettler27 — APPROVED** [review]
The code coverage is now improved, great addition

**Review by AlexandreMessmer — APPROVED** [review]

File: app/src/debug/AndroidManifest.xml (hunk 1 above)
Comment (mbouilloud) [reply app/src/debug/AndroidManifest.xml]: Done.

**Pull request merged**